src/console_gui.cpp

```diff
@@ -258,8 +303,18 @@ struct IConsoleWindow : Window
 				IConsoleCmdExec("clear");
 				break;
 
+			case WKC_TAB:
+				if (_iconsole_tab_completion.AutoComplete()) {
+					this->SetDirty();
+				}
+				break;
+
 			default:
-				if (_iconsole_cmdline.HandleKeyPress(key, keycode) != HKPR_NOT_HANDLED) {
+				HandleKeyPressResult handle_result = _iconsole_cmdline.HandleKeyPress(key, keycode);
```

Comment:
It mostly works, but it's good practice to use braces when introducing variable declarations into a case block

Reply:
Done